parent/pom.xml

```diff
@@ -493,6 +493,20 @@
             <version>${version.hibernate.entitymanager}</version>
          </dependency>
          <dependency>
+            <groupId>org.jboss.shrinkwrap.resolver</groupId>
+            <artifactId>shrinkwrap-resolver-bom</artifactId>
+            <version>${version.shrinkwrapResolver}</version>
+            <type>pom</type>
+            <scope>import</scope>
+         </dependency>
+         <!-- This is a work around for https://issues.jboss.org/browse/SHRINKRES-238 -->
+         <!-- After fixing the issue above, the dependency below can be safely removed -->
```

Comment:
By uber-jar I meant "org.infinispan:infinispan-cdi-embedded" in the test -- I mistook it for one of our uber-jars, because that's what I thought SHRINKRES-238 was about. Sorry about that!

I was also puzzled by `<scope>import</scope>`, I don't think I ever saw this being used before. I see in your link that was the recommended way of adding the dependency with Arquillian prior to 1.1.0.Final, but I guess it's fine to keep it that way.